data/workshops/en/2017.md

```diff
@@ -0,0 +1,226 @@
+    slides: https://speakerdeck.com/avsm/ocaml-platform-2017
+    link: workshop/2017/slides__2017__anil-madhavapeddy__the-state-of-the-ocaml-platform-september-2017.pdf
+  - title: "Wodan: a pure OCaml, flash-aware filesystem library" 
+    authors: 
+      - Gabriel de Perthuis 
+    link: workshop/2017/extended-abstract__2017__gabriel-de-perthuis__wodan-a-pure-ocaml-flash-aware-filesystem-library.pdf
+program_committee: 
+  - name: Ashish Agarwal
+    affiliation: Solvuu, USA
+  - name: François Bobot
+    affiliation: CEA, France
+  - name: Frédéric Bour
+    affiliation: OCaml Labs, France
+  - name: Cristiano Calcagno
+    affiliation: Facebook, UK
+  - name: Louis Gesbert
+    affiliation: OcamlPro, France
+  - name: Sébastien Hinderer
+    affiliation: INRIA, France
+  - name: Atsushi Igarashi
+    affiliation: Kyoto University, Japan
+  - name: Oleg Kiselyov
+    affiliation: Tohoku University, Japan
+  - name: Julia Lawall
+    affiliation: INRIA/LIP6, France
+  - name: Sam Lindley
+    affiliation: The University of Edinburgh, UK
+  - name: Louis Mandel
+    affiliation: IBM Research, USA
+  - name: Zoe Paraskevopoulou 
+    affiliation: Princeton University, USA
+  - name: Gabriel Scherer
+    affiliation: Northeastern University, USA
+organising_committee: []
+---
+
+OCaml 2017 will open with an invited talk by three frequent
+contributors that recently became maintainers of the OCaml
+implementation: David Allsopp
+([video](https://www.youtube.com/watch?v=10OQHsnyg64&index=2&list=TLGGpj_CrU7rr7MxMjAxMjAxOA)),
+Florian Angeletti
+([video](https://www.youtube.com/watch?v=HOfdGDSypP4&list=TLGGpj_CrU7rr7MxMjAxMjAxOA&index=5)),
+and Sébastien Hinderer
+([video](https://www.youtube.com/watch?v=SvnyQWZkHS8&list=TLGGpj_CrU7rr7MxMjAxMjAxOA&index=1)).
+
+Due to the high number of high-quality submissions, we had to have
+more posters than in previous editions to fit a one-day
+schedule. 
+
+
+The following works will be presented as posters.
+
+- _ocamli: Interpreted OCaml_ —
+  John Whitington  
+  [extended abstract](http://www.cs.le.ac.uk/people/jw642/ocamlworkshop.pdf)
+  ([local copy](extended-abstract__2017__john_whitington__ocamli-interpreted-ocaml.pdf))
+
+- _mSAT: An OCaml SAT Solver_ —
+  Bury Guillaume  
+  [extended abstract](https://gbury.eu/public/papers/icfp2017_msat.pdf)
+  ([local copy](extended-abstract__2017__guillaume-bury__msat-an-ocaml-sat-solver.pdf))
+
+- _Tyre – Typed Regular Expressions_ —
+  Gabriel Radanne  
+  ([extended abstract](https://www.irif.fr/~gradanne/papers/tyre/abstract.pdf)
+   ([local copy](extended-abstract__2017__gabriel-radanne__tyre-typed-regular-expressions.pdf)))
+
+- _Jbuilder: a modern approach to OCaml development_ —
+  Jeremie Dimino, Mark Shinwell  
+  ([local copy](extended-abstract__2017__jeremie-dimino_mark-shinwell__jbuilder-a-modern-approach-to-ocaml-development.pdf))
+
+
+Call for presentations (past)
+=============================
```

Comment:
Could we replace this with standard markdown titles (i.e. the style of the titles we use for the other data)?

Reply:
Sure thing